src/NHibernate/Driver/OracleClientDriver.cs

```diff
@@ -8,6 +9,9 @@ namespace NHibernate.Driver
 	/// <summary>
 	/// A NHibernate Driver for using the Oracle DataProvider.
 	/// </summary>
+	// Since v5.3
+	// Deprecated by Microsoft: https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/oracle-and-adonet
+	[Obsolete("Use OracleManagedDataClientDriver or OracleDataClientDriver driver instead.")]
 	public class OracleClientDriver : ReflectionBasedDriver
```

Comment:
Let's not deprecate it yet.

Reply:
If we do not deprecate it before a major release which would switch to .Net Framework 5, that would mean a drop of this without prior warning through a deprecation.

What is the trouble with deprecating it? Forcing people to add directives for disabling deprecation warning? It should be very infrequent, since this is usually loaded through configuration, and so not seen by the compiler.

At "worst", a NHibernate warning log will be emitted. (I know in some cases NHibernate does detect and warn about obsolete classes used through configuration, but I have not checked if this is the case for drivers.)

But as this is not the subject of this PR, why not doing this in another PR anyway.

Reply:
Ok